src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
 
-He reached out his hand for a fist bump.
+(set: $cartPartsSeen to it +1)(set: $engineSeen to true)
 
-[[I reached my fist out to reciprocate.|The Gutter - Outside - old-school juggalos - part 5]]
-[[I quickly blurted out, “You seen another little ‘lette around here recently? Maybe in trouble? Her name’s Anna. She’s in a bad way.”|The Gutter - Outside - old-school juggalos - part 6]]
+::The Gutter - Jar
+A huge mason jar hung on a rusty nail right outside the front door. Above it in individual sticker letters it read:
 
-::The Gutter - Outside - old-school juggalos - part 5 <679,1114>
-“You know,” Large said, “I always hated it when my fucking old man would rattle on and hit me with advice, but shit. Guess I’m older than he was now.” He lit a cigarette and inhaled. “Always keep the family at the front of your mind.”
+(text-style: "outline")[Leave Y ur Mind Here]
 
-I tried to nod back, conveying a wisdom that I didn’t have yet, and we parted ways.
+It had hung there empty for as long as I could remember.
 
-As I walked back towards the front of the Gutter
+[[I kept looking around|The Gutter - Outside hub]].
 
->> [I dodged a hole in the ground.]<dodgeTrigger|
->> [my dumb ass tripped over a hole in the ground.]<tripTrigger|
+(set: $jarSeen to true)
 
-|dodgeSection)[Hell yeah.]
-|tripSection)[I pushed myself up before the dudes could help me. I believed the shit about family, but tonight I could help myself.]
+::The Gutter - Name prompt inside
+{ (set: $playerName to (prompt: "Enter name:", ""))
+(if: (text: $playerName)'s length > 0)[Right, my name is $playerName. [[Moving on|The Gutter - Inside intro]].]
+}(else:)[No, [[that's not right|The Gutter - Name prompt inside]].]
 
-[[Anyway...|The Gutter - Outside]]
+(set: $dissolve to (transition: 'dissolve') + (transition-time: 3s))
+(set: $hatchet to true)(set: $multitool to true)
 
-(click: ?dodgeTrigger)[(show: ?dodgeSection)]\
-(click: ?tripTrigger)[(show: ?tripSection)]
+::The Gutter - Inside intro
+Inside the Gutter, the usual piss-odor of Gatherville faded away under a heavy cloud of cigarette, hookah, and bong smoke. It was thick enough to sting my eyes. I could barely make out anyone inside.
 
-::The Gutter - Outside - old-school juggalos - part 6 <821,1114>
-“Aw shit,” Large raised his eyebrows, “sorry to hear that. Me and Kyle been bullshitting out here all night. Haven’t seen anything like that.”
+[I waved my hand in front of my face to clear the air.]<waveHand|
 
-Kyle piped up. “Me and Ben’ll keep an eye out though for sure!”
+[<span class="blur">The crowd was light, so no one even bothered moving the two drunk teens making out right on top of the torn felt of the pool table. The Jukebox for this evening was braying some new tune of his at a weirdly low volume in back of the room. The bartender looked a mix of bored and annoyed as he moved between serving a trio of barflies, who were spewing smoke and holding court on their stools.
```

Comment:
I noticed jukebox is capitalized throughout, but to my knowledge it's not a proper noun. We should probably replace all these with lower-case "jukebox."

Reply:
Jukebox is capped to emphasize that this is a special role, rather than a machine. This is largely to tip the player off. I can still see an argument for not capping, because most jobs aren't proper nouns either, but I do think capping helps get across the new use for the term.